Tools/TestWebKitAPI/PlatformUtilities.h

```diff
@@ -40,6 +40,14 @@ OBJC_CLASS NSDictionary;
 typedef double NSTimeInterval;
 #endif
 
+#if PLATFORM(COCOA)
+OBJC_CLASS NSImage;
+OBJC_CLASS NSWindow;
+OBJC_CLASS UIImage;
+OBJC_CLASS UIWindow;
+typedef struct CGImage *CGImageRef;
```

Comment:
Not that it matters, but optionally:
```suggestion
using CGImageRef = CGImage*;
```

Reply:
Will do!